src/towncrier/_git.py

```diff
@@ -10,18 +10,25 @@
 import click
 
 
-def remove_files(fragment_filenames: list[str], answer_yes: bool) -> None:
+def remove_files(
+    fragment_filenames: list[str], answer_yes: bool, answer_keep: bool
+) -> None:
     if not fragment_filenames:
         return
 
     if answer_yes:
         click.echo("Removing the following files:")
+    elif answer_keep:
+        click.echo("Keeping the following files:")
     else:
         click.echo("I want to remove the following files:")
 
     for filename in fragment_filenames:
         click.echo(filename)
 
+    if answer_keep and not answer_yes:
+        return
```

Comment:
Can we can "answer_yes == True" here ?

I was expecting that this is validated in the caller code.
By handling this branch, somebody might think that this is a valid use-case.

```suggestion
    if answer_keep:
        # Not proceeding with the removal of the files.
        return
```

Reply:
Hmm.... you're right